plugins/network-elements/tungsten/src/main/java/org/apache/cloudstack/network/tungsten/api/command/GetLoadBalancerSslCertificateCmd.java

```diff
@@ -0,0 +1,82 @@
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.apache.cloudstack.network.tungsten.api.command;
+
+import com.cloud.exception.ConcurrentOperationException;
+import com.cloud.exception.InsufficientCapacityException;
+import com.cloud.exception.NetworkRuleConflictException;
+import com.cloud.exception.ResourceAllocationException;
+import com.cloud.exception.ResourceUnavailableException;
+import com.cloud.network.lb.LoadBalancingRule;
+import com.cloud.network.lb.LoadBalancingRulesManager;
+import com.cloud.user.Account;
+import com.cloud.utils.exception.CloudRuntimeException;
+import org.apache.cloudstack.api.APICommand;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.BaseCmd;
+import org.apache.cloudstack.api.Parameter;
+import org.apache.cloudstack.api.ServerApiException;
+import org.apache.cloudstack.context.CallContext;
+import org.apache.cloudstack.network.tungsten.api.response.TlsDataResponse;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.log4j.Logger;
+
+import javax.inject.Inject;
+
+@APICommand(name = "getLoadBalancerSslCertificate", description = "get load balancer certificate", responseObject =
+    TlsDataResponse.class, requestHasSensitiveInfo = false, responseHasSensitiveInfo = false)
+public class GetLoadBalancerSslCertificateCmd extends BaseCmd {
+    public static final Logger s_logger = Logger.getLogger(GetLoadBalancerSslCertificateCmd.class.getName());
+    private static final String s_name = "response";
+
+    @Inject
+    private LoadBalancingRulesManager _lbMgr;
+
+    @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = TlsDataResponse.class, required = true,
+        description = "the ID of Lb")
+    private Long id;
+
+    @Override
+    public void execute() throws ResourceUnavailableException, InsufficientCapacityException, ServerApiException,
+        ConcurrentOperationException, ResourceAllocationException, NetworkRuleConflictException {
+        LoadBalancingRule.LbSslCert lbSslCert = _lbMgr.getLbSslCert(id);
+        if (lbSslCert != null) {
+            TlsDataResponse tlsDataResponse = new TlsDataResponse();
+            tlsDataResponse.setCrt(Base64.encodeBase64String(lbSslCert.getCert().getBytes()));
+            tlsDataResponse.setKey(Base64.encodeBase64String(lbSslCert.getKey().getBytes()));
+            tlsDataResponse.setResponseName(s_name);
+            tlsDataResponse.setObjectName("data");
+            setResponseObject(tlsDataResponse);
+        } else {
+            throw new CloudRuntimeException("can not get tls data");
+        }
+    }
+
+    @Override
+    public String getCommandName() {
+        return s_name;
```

Comment:
ususally this is 
```suggestion
        return s_name.toLower() + "response";
```
there is a lof of Cmd.java classes here that could be more conformant is this way.